simulate-event.js

```diff
@@ -281,6 +304,20 @@ module.exports = function (element, type, options) {
 
   event = extend(document.createEvent(eventType), options);
 
+  // Handle differences between `initKeyboardEvent` and `initKeyEvent`.
+  if (initEvent === 'initKeyboardEvent') {
+    if (event[initEvent] === void 0) {
+      initEvent = 'initKeyEvent';
+    } else if (!('modifiersList' in options)) {
```

Comment:
Ok, all good. Do we want to preserve the old list at all? Probably doesn't matter, I'm good with this now (maybe leave a comment for the next guy though).